### ISSUE TITLE
Send First Deliverables

### ISSUE THREAD

**Issue by joanafbrito** [joanafbrito/Marvel2#6]
- [x]  Create a DM with both instructors
 - [x]  Your niche audience (be as specific and narrow as possible)
 - [x] Your chosen API
-  [x] A list of your app’s MVP features
 - [x] Optional: a list of your app’s nice-to-have features
 - [ ] A backup API & audience
Once your project MVP is approved (instructors will help you narrow down your MVP to a reasonable scope)
